lib/timeline/Core.js

```diff
@@ -51,6 +51,8 @@ Core.prototype._create = function (container) {
   this.dom.shadowTopRight       = document.createElement('div');
   this.dom.shadowBottomRight    = document.createElement('div');
   this.dom.rollingModeBtn       = document.createElement('div');
+  this.dom.leftZone             = document.createElement('div');
```

Comment:
I have a problem with these names. They are too generic. I think they should be renamed to: `panningZoneLeft` and `panningZoneRight`